src/js/App/Header/InsightsAbout.js

```diff
@@ -73,7 +73,9 @@ class InsightsAbout extends Component {
                 { name: 'Compliance', path: 'apps/compliance/app.info.json', version: 'N/A' },
                 { name: 'Cost Management', path: 'apps/cost-management/app.info.json', version: 'N/A' },
                 { name: 'Insights', path: 'apps/insights/app.info.json', version: 'N/A' },
-                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' }
+                { name: 'System Comparison', path: 'apps/drift/app.info.json', version: 'N/A' },
+                //concerned about this bit
+                { name: 'Migrations', path: 'migrations/migration-analytics/app.info.json', version: 'N/A' }
```

Comment:
I think this path is actually going to be `apps/migration-analytics/app.info.json`

They're in beta right now, so you can see the path:
https://ci.cloud.paas.psi.redhat.com/beta/apps/migration-analytics/app.info.json

Reply:
Thanks! I was concerned about that bit